FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/ApiClientRepository.kt

```diff
@@ -51,78 +52,117 @@ import java.util.concurrent.TimeUnit
+        ApiHelper.getInstance(context).retrofit.create(RetrofitApiServiceInterface::class.java)
       getResult {
-        apiService.submitPrimaryEmailPubKey(
+        retrofitApiService.submitPrimaryEmailPubKey(
           authorization = "Bearer $idToken",
           email = email,
           pubKey = pubKey
         )
       }
     }
 
-  override suspend fun submitPubKeyWithConditionalEmailVerification(
+  /**
+   * @param context Interface to global information about an application environment.
+   * @param email For this email address will be applied changes.
+   * @param pubKey A new public key.
+   * @param clientConfiguration An instance of [ClientConfiguration]. We have to check if submitting pub keys is allowed.
+   */
+  suspend fun submitPubKeyWithConditionalEmailVerification(
     context: Context,
     email: String,
     pubKey: String,
-    clientConfiguration: ClientConfiguration?
+    clientConfiguration: ClientConfiguration? = null
   ): Result<SubmitPubKeyResponse> =
     withContext(Dispatchers.IO) {
       if (clientConfiguration?.canSubmitPubToAttester() == false) {
         return@withContext Result.exception(
           IllegalStateException(context.getString(R.string.can_not_replace_public_key_at_attester))
         )
       }
-      val apiService = ApiHelper.getInstance(context).retrofit.create(ApiService::class.java)
-      getResult { apiService.submitPubKeyWithConditionalEmailVerification(email, pubKey) }
+      val retrofitApiService =
+        ApiHelper.getInstance(context).retrofit.create(RetrofitApiServiceInterface::class.java)
+      getResult { retrofitApiService.submitPubKeyWithConditionalEmailVerification(email, pubKey) }
     }
 
-  override suspend fun postWelcomeMessage(
+  /**
+   * @param context Interface to global information about an application environment.
+   * @param idToken OIDC token.
+   * @param model An instance of [WelcomeMessageModel].
+   */
+  suspend fun postWelcomeMessage(
     context: Context,
     idToken: String,
     model: WelcomeMessageModel,
   ): Result<WelcomeMessageResponse> =
     withContext(Dispatchers.IO) {
-      val apiService = ApiHelper.getInstance(context).retrofit.create(ApiService::class.java)
-      getResult { apiService.postWelcomeMessage(authorization = "Bearer $idToken", body = model) }
+      val retrofitApiService =
+        ApiHelper.getInstance(context).retrofit.create(RetrofitApiServiceInterface::class.java)
+      getResult {
+        retrofitApiService.postWelcomeMessage(
+          authorization = "Bearer $idToken",
+          body = model
+        )
+      }
     }
 
-  override suspend fun pubLookup(
-    requestCode: Long,
+  /**
+   * @param requestCode A unique request code for this call
+   * @param context     Interface to global information about an application environment.
+   * @param email       A user email.
+   * @param clientConfiguration    Contains client configurations.
+   */
+  suspend fun pubLookup(
+    requestCode: Long = 0L,
```

Comment:
What is the `requestCode` for and what does `0` mean?

Reply:
I've removed this code. We used to use it but now do not.